workflows4s-example/src/main/scala/workflows4s/example/docs/doobie/DatabaseExample.scala

```diff
@@ -3,42 +3,48 @@ package workflows4s.example.docs.doobie
 import cats.effect.IO
 import doobie.util.transactor.Transactor
 import workflows4s.doobie.postgres.PostgresWorkflowStorage
-import workflows4s.doobie.{ByteCodec, DatabaseRuntime, WorkflowStorage}
-import workflows4s.runtime.WorkflowInstance
-import workflows4s.runtime.instanceengine.WorkflowInstanceEngine
-import workflows4s.wio.{WCState, WorkflowContext}
+import workflows4s.doobie.{ByteCodec, WorkflowStorage}
+import workflows4s.wio.WorkflowContext
 
 import scala.annotation.nowarn
 
-@nowarn("msg=unused local definition")
+@nowarn
 object DatabaseExample {
 
-  object MyWorkflowCtx extends WorkflowContext {
-    sealed trait State
-    case class InitialState() extends State
-    sealed trait Event
-  }
+  // Database runtimes use IO effect type directly
+  // Below shows the conceptual pattern - see full example in tests
 
-  import MyWorkflowCtx.*
-  {
-    // doc_start
-    val workflow: WIO.Initial           = ???
-    val initialState: State             = ???
-    val transactor: Transactor[IO]      = ???
-    val storage: WorkflowStorage[Event] = ???
-    val engine: WorkflowInstanceEngine  = ???
-    val templateId                      = "my-workflow"
-
-    val runtime: DatabaseRuntime[Ctx]                      = DatabaseRuntime.create(workflow, initialState, transactor, engine, storage, templateId)
-    val wfInstance: IO[WorkflowInstance[IO, WCState[Ctx]]] = runtime.createInstance("1")
-    // doc_end
+  // doc_start
+  // Define your workflow context
+  trait MyCtx extends WorkflowContext {
+    type State = String
+    type Event = String
   }
+  type Ctx = MyCtx
+
+  val transactor: Transactor[IO]           = ???
+  val storage: WorkflowStorage[IO, String] = ???
+  val templateId                           = "my-workflow"
+
+  // For actual usage, see workflows4s-doobie tests
+  // DatabaseRuntime.create takes:
+  // - workflow: WIO.Initial[IO, Ctx]
+  // - initialState: WCState[Ctx]
+  // - transactor: Transactor[IO]
+  // - engine: WorkflowInstanceEngine[IO]
+  // - eventCodec: ByteCodec[WCEvent[Ctx]]
+  // - templateId: String
+
+  // After creation:
+  // val runtime: DatabaseRuntime[Ctx] = DatabaseRuntime.create(...)
+  // val wfInstance: IO[WorkflowInstance[IO, WCState[Ctx]]] = runtime.createInstance("1")
+  // doc_end
```

Comment:
what?  The whole purpose of this snippet was to show compling example. We shouldnt be pointing users to tests instead...